w3af/plugins/output/xml_file/finding.tpl

```diff
@@ -1,4 +1,4 @@
-<vulnerability id="{{ id_list }}" method="{{ http_method | escape_attr }}" name="{{ name | escape_attr }}" plugin="{{ plugin_name }}" severity="{{ severity }}" url="{{ url | escape_attr }}" var="{{ var | escape_attr }}">
+<vulnerability id="{{ id_list }}" method="{{ http_method | escape_attr }}" name="{{ name | escape_attr }}" plugin="{{ plugin_name }}" severity="{{ severity }}" url="{{ url | escape_attr }}" var="{{ var | escape_attr }}" vulndb_id="{{ vulndb_id }}">
```

Comment:
Maybe `vulndb_id={{ vulndb_id }}` instead of `vulndb_id="{{ vulndb_id }}"`? Those IDs are ints. (just a detail)